keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/Deregister.java

```diff
@@ -0,0 +1,18 @@
+package net.yslibrary.android.keyboardvisibilityevent;
+
+import android.view.ViewTreeObserver;
+
+/**
+ * @author Anoop S S
+ *         anoopvvs@gmail.com
+ *         on 28/02/2017
+ */
+
+public interface Deregister {
```

Comment:
Any reason you prefer `deregister` over `unregister`?
I think Android people are more familiar with `register - unregister` pair than `register - deregister`(such as `Context#registerReceiver` and `Context#unregisterReceiver`).

Also, class/interface name should be noun.